### PR TITLE
loadout tidbits

### PR DESCRIPTION
**Pull request by peeesh**
i know its TINY but seriously gib cigar case. oh, skrellian implant too i hope. might put in more junk too, so dont merge just yet (or, yknow, ever, if youre rude)

### CODE REVIEW

**Comment by peeesh** [comment]
cool to merge now
oh i forgot to mention new rings too

**Comment by peeesh** [comment]
nO DONT IM PUTTING AR GLASSES IN ONE SEC

**Comment by peeesh** [comment]
jesus fucking christ boys, just merge whenever. lore is done n everything

**Comment by peeesh** [comment]
dOOONE aaaaaaaaaa

File: code/modules/client/preference_setup/loadout/loadout_gloves.dm [hunk 1]
```diff
@@ -98,6 +98,9 @@
 	ringtype["ring, glass"] = /obj/item/clothing/gloves/ring/material/glass
 	ringtype["ring, wood"] = /obj/item/clothing/gloves/ring/material/wood
 	ringtype["ring, plastic"] = /obj/item/clothing/gloves/ring/material/plastic
+	ringtype["ring, uranium"] = /obj/item/clothing/gloves/ring/material/uranium
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/loadout_gloves.dm]: If I recall this gives slowly rads and toxin harm...

Comment (peeesh) [reply code/modules/client/preference_setup/loadout/loadout_gloves.dm]: i tested it locally, wore it for solid 30-60 minutes, no changes on healthdoll to show "hey youre dying"

**Pull request merged**